CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKFPSRenderable.java

```diff
@@ -1019,6 +1053,16 @@ private int CreatePipelines() {
                 colorBlending.pAttachments(colorBlendAttachment);
                 colorBlending.blendConstants(stack.floats(0.0f, 0.0f, 0.0f, 0.0f));
 
+                // ===> DYNAMIC PROPERTIES CREATION <===
```

Comment:
Did you test the FPS moves to the right location following a resize?

Reply:
yep